docs/source/tutorials/base/how_to_train/detection.rst

```diff
@@ -0,0 +1,385 @@
+                            --load-weights outputs/openvino/openvino.xml
+                            --save-performance outputs/openvino/performance.json
+  
+  ...
+  2023-01-10 06:24:50,382 | INFO : Start OpenVINO inference
+  2023-01-10 06:24:54,943 | INFO : OpenVINO inference completed
+  2023-01-10 06:24:54,944 | INFO : Start OpenVINO metric evaluation
+  2023-01-10 06:24:55,117 | INFO : OpenVINO metric evaluation completed
+  Performance(score: 0.5487693710118504, dashboard: (1 metric groups))
+
+
+*************
+Optimization
+*************
+
+1. We can further optimize the model with ``otx optimize``. It uses NNCF or POT depending on the model format.
+
+``NNCF`` optimization is used for trained snapshots in a framework-specific format such as checkpoint (pth) file from Pytorch. It starts accuracy-aware quantization based on the obtained weights from the training stage. Generally, we will see the same output as during training.
+
+``POT`` optimization is used for models exported in the OpenVINO™ IR format. It decreases floating-point precision to integer precision of the exported model by performing the post-training optimization.
+
+The function results with the following files, which could be used to run :doc:`otx demo <../demo>`:
+
+- ``confidence_threshold``
+- ``config.json``
+- ``label_schema.json``
+- ``openvino.bin``
+- ``openvino.xml``
+
+To learn more about optimization, refer to `NNCF repository <https://github.com/openvinotoolkit/nncf>`_.
+
+2. Command example for optimizing a PyTorch model (`.pth`) with OpenVINO NNCF. We can also simplify the commanline by adding ``--data data.yaml`` parameter instead of specifying training and validation paths there.
+
+.. code-block::
+
+  (detection) ...$ otx optimize otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml 
+                                --train-ann-files data/wgisd/coco_annotations/train_bbox_instances.json 
+                                --train-data-roots  data/wgisd/data
+                                --val-ann-files data/wgisd/coco_annotations/test_bbox_instances.json 
+                                --val-data-roots data/wgisd/data 
+                                --load-weights outputs/weights.pth
+                                --save-model-to outputs/nncf
+                                --save-performance outputs/nncf/performance.json
+
+  ...
+
+  2023-01-17 06:46:08,208 | INFO : run task done.
+  2023-01-17 06:46:08,618 | INFO : Inference completed
+  2023-01-17 06:46:08,618 | INFO : called evaluate()
+  2023-01-17 06:46:08,829 | INFO : F-measure after evaluation: 0.5446735395189003
+  2023-01-17 06:46:08,829 | INFO : Evaluation completed
+  Performance(score: 0.5446735395189003, dashboard: (1 metric groups))
+
+
+3. Command example for optimizing OpenVINO model (.xml) with OpenVINO POT. We can also simplify the commanline by adding ``--data data.yaml`` parameter instead of specifying training and validation paths there.
```

Comment:
```suggestion
3. Command example for optimizing OpenVINO™ model (.xml) with OpenVINO™ POT. We can also simplify the commanline by adding ``--data data.yaml`` parameter instead of specifying training and validation paths there.
```